### ISSUE TITLE
the PyTorch classifier has not ability to change the learning rate.

### ISSUE THREAD

**Issue by mcdonaldnj304** [Trusted-AI/adversarial-robustness-toolbox#896]
**Is your feature request related to a problem? Please describe.**
My problem if for training a model from scratch one should need to change the learning rate as the model moves through the assigned epochs. PyTorch does this through a few different methods like Cosine Annealing, and a learning rate scheduler. 

**Describe the solution you'd like**
PyTorch has the lr_scheduler built into torch.optim, however there is nothing to allow this in the __init__() of the PyTorch Classifier 

**Describe alternatives you've considered**

**Additional context**

**Comment by beat-buesser** [comment]
Hi @mcdonaldnj304 Thank you very much for your interest in ART! With ART you can also train the model outside of the `art.estimators` and provide the trained model to argument `model` of the estimator, this gives you all the flexibility of PyTorch. Would that work in your case?
